nextflow_schema.json

```diff
@@ -258,6 +258,28 @@
                     "default": "prefix",
                     "fa_icon": "fas fa-list-ol",
                     "help_text": "Prefix is highly recommended. Only in case an external tool marked decoys with a suffix, e.g. `sp|Q12345|ProteinA_DECOY` change this parameter to suffix."
```

Comment:
I think we should force here to always been Prefix if we know that Percolator will fail. Is actually confusing. We should not expose the prefix and suffix option.

Reply:
the thing is: if you have a decoy DB from somewhere and it has these suffixes, you would not be able to use the pipeline. At least you could run it without Percolator. Would probably be a quick fix in percolator anyway.